addon/components/base/bs-form/element.js

```diff
@@ -617,11 +639,16 @@ export default FormGroup.extend({
    * @type string
    * @private
    */
-  validation: computed('hasCustomError', 'hasErrors', 'hasWarnings', 'hasValidator', 'showValidation', 'isValidating', 'disabled', function() {
+  validation: computed('hasCustomError', 'hasErrors', 'hasCustomWarning', 'hasWarnings', 'hasValidator', 'showValidation', 'isValidating', 'disabled', function() {
     if (!this.get('showValidation') || !this.get('hasValidator') || this.get('isValidating') || this.get('disabled')) {
       return null;
+    } else if (this.get('showOwnValidation') || this.get('showAllValidations')) {
```

Comment:
Why do we need this added if-statement, compared to the previous implementation? 
And `showOwnValidation` and `showAllValidations` are not declared as dependent keys of the computed property, if we really need this...